AsyncDisplayKit/Details/ASEnvironment.h

```diff
@@ -157,6 +157,10 @@ ASDISPLAYNODE_EXTERN_C_END
   ASDN::MutexLocker l(lock);\
   ASEnvironmentTraitCollection oldTraits = self.environmentState.environmentTraitCollection;\
   [super setEnvironmentState:environmentState];\
+\
+   /* Extra Trait Collection Handling */\
```

Comment:
@maicki it looks like there is an extra space in front of this line compared to the next line (maybe?).

Thanks a lot for debugging this!